src/main/perl/Reporter.pm

```diff
@@ -202,6 +202,12 @@ sub _print
 sub report
 {
     my $self = shift;
+    my $i=0;
+    # Ensure that there is no undefined arg: replace by '' if any
+    while ( $i < scalar(@_)) {
+      $_[$i] = '' if !defined($_[$i]);
```

Comment:
i'm not in favour of using `''`, better to use `<undef>` or something that is obvious (as this is actually exposing a bug in whatever calls the reporter).